src/tremor_sys.rs

```diff
@@ -7,8 +7,8 @@ extern crate libc;
 use libc::{c_int, c_long, c_char, c_void};
 
 pub use ogg_sys::ogg_int64_t;
-pub type ogg_int32_t = libc::int32_t;
```

Comment:
This is Linuxism. Please keep int32_t .

Reply:
It's simply deprecated: http://rust-lang.github.io/libc/x86_64-unknown-hermit/libc/type.int32_t.html

Reply:
I see, not a C-language type. Then it is probably fine. Sorry for the noise.